integrations/huggingface-transformers/tutorials/question-answering/question_answering-cli.md

```diff
@@ -0,0 +1,372 @@
+
+### Pre-Sparsified BERT
+
+SparseZoo, Neural Magic's open source repository of pre-sparsified models, contains a 90% pruned version of BERT, which has been sparsified on the upstream Wikipedia and BookCorpus datasets with the masked language modeling objective. [Check out the model card](https://sparsezoo.neuralmagic.com/models/nlp%2Fmasked_language_modeling%2Fobert-base%2Fpytorch%2Fhuggingface%2Fwikipedia_bookcorpus%2Fpruned90-none). We will use this model as the starting point for the transfer learning process.
+
+**Let's dive in!**
+
+## Installation
+
+Install SparseML via `pip`.
+
+```bash
+pip install sparseml[torch]
+```
+
+## Sparse Transfer Learning onto MNLI (GLUE Task)
+
+SparseML's CLI enables you to kick-off sparsification workflows with various utilities like creating training pipelines, dataset loading, checkpoint saving, metric reporting, and logging handled for you. 
+
+All we have to do is pass a couple of key arguments: 
+- `--model_name_or_path` specifies the starting checkpoint to load for training
+- `--task` specifies a glue dataset to train with 
+- `--recipe` specifies path a recipe to use to apply sparsification algorithms or sparse transfer learning to the model. For Sparse Transfer Learning, we will use a recipe that instructs SparseML to maintain sparsity during the training process and to apply quantization over the final few epochs. 
+
+### Run Transfer Learning
+
+We will fine-tune a [90% pruned version of BERT](zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none) onto MNLI.
+
+Run the following:
+```bash
+sparseml.transformers.train.question_answering \
+  --dataset_name squad \
+  --model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none \
+  --recipe zoo:nlp/question_answering/obert-base/pytorch/huggingface/squad/pruned90_quant-none \
+  --distill_teacher zoo:nlp/question_answering/obert-base/pytorch/huggingface/squad/base-none \
+  --output_dir obert_base_pruned90_quant_squad \
+  --do_train --do_eval --evaluation_strategy epoch --logging_steps 1000 --save_steps 1000 \
+  --per_device_train_batch_size 8 --per_device_eval_batch_size 32 --gradient_accumulation_steps 2 --preprocessing_num_workers 32 \
+  --max_seq_length 384 --doc_stride 128 \
+  --seed 42
+```
+
+Let's discuss the key arguments:
+- `--dataset_name squad` instructs SparseML to download and fine-tune onto the SQuAD dataset. The data is downloaded from the Hugging Face hub. You can pass any extractive QA dataset from the Hugging Face hub, provided it conforms to the SQuAD format (see below for details).
```

Comment:
It is not clear where this below is. Maybe make it explicit that it's the `Squadshifts Dataset Inspection` section. Same comments for other places with "See below"